microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -112,7 +114,38 @@ if (sendMessageId) {
 }
 ```
 
-4. Make your stars.jpeg available from any browser tab
+#### [Manifest V3](#tab/v3)
+
+```javascript
+const sendMessageId = document.getElementById("sendmessageid");
+if (sendMessageId) {
+    sendMessageId.onclick = function() {
+        chrome.tabs.query({ active: true, currentWindow: true }, function(tabs) {
+            chrome.tabs.sendMessage(
+                tabs[0].id,
+                {
+                    url: chrome.runtime.getURL("images/stars.jpeg"),
+                    imageDivId: `${guidGenerator()}`,
+                    tabId: tabs[0].id
+                },
+                function(response) {
+                    window.close();
+                }
+            );
+            function guidGenerator() {
+                const S4 = function () {
+                    return (((1 + Math.random()) * 0x10000) | 0).toString(16).substring(1);
+                };
+                return (S4() + S4() + "-" + S4() + "-" + S4() + "-" + S4() + "-" + S4() + S4() + S4());
+            }
+        });
+    };
+}
+```
+
+---
+
+4. Make your `stars.jpeg` available from any browser tab
 
 You're probably wondering why, when you pass the `images/stars.jpeg` must you use the `chrome.extension.getURL` chrome Extension API instead of just passing in the relative URL without the extra prefix like in the previous section.  By the way, that extra prefix, returned by `getUrl` with the image attached looks something like the following.
```

Comment:
This V2-specific word|sentence|paragraph needs techreview in future PR.  chrome.extension.getURL is V2-specific.  @ashishnirvana1 @eclipse439